src/Microsoft.Deployment.DotNet.Releases/src/Product.cs

```diff
@@ -130,12 +134,21 @@ public SupportPhase SupportPhase
             private set;
         } = SupportPhase.Unknown;
 
+        /// <summary>
```

Comment:
So, this is just adding some symmetry to the API?

Reply:
Yes. I actually always believed that this would have been better as a read-only property rather than a function on the type. 

When I first switched over to [consuming this library](https://github.com/dotnet/versionsweeper/pull/17), I did a lot of testing and debugging. I initially had an extension method on `Product`, but figured it would be nicely if the object represented what I wanted. Having a simple `bool` that states that it `IsSupported` seems to be a nicety. I was tempted to propose that we remove the `IsOutOfSupport()` function, as properties are cleaner IMO here.